lib/ably/realtime/presence/members_map.rb

```diff
@@ -228,13 +228,11 @@ def setup_event_handlers
 
       def enter_local_members
         local_members.values.each do |member|
-          local_client_id = member.client_id || client.auth.client_id
```

Comment:
Why is this commit in this PR? It doesn't seem related. It's already in https://github.com/ably/ably-ruby/pull/410.

Reply:
It's not directly related, though we had mentioned to fix it as a part of this PR -> https://github.com/ably/ably-ruby/pull/410#discussion_r1650794411.
I will revert this for the time being to better understand the PR.

Reply:
Yeah, it doesn't seem related to the current PR to me.

Reply:
reverted https://github.com/ably/ably-ruby/pull/418/commits/91e5f29b8324196225b113ce81b4fdd441780b67